src/externalized/AmmoTypeModel.cc

```diff
@@ -1,11 +1,11 @@
 #include "AmmoTypeModel.h"
 #include "JsonObject.h"
+#include <stdint.h>
```

Comment:
Why not cstdint?

Reply:
Because the C++ standard only requires cstdint to put these names in the std namespace; putting them in the global namespace  (where string_theory needs them) is optional.